content/en/docs/usage/certificate.md

```diff
@@ -167,37 +167,81 @@ If you would prefer the `Secret` to be deleted automatically when the `Certifica
 
 ## Renewal
 
+
 `cert-manager` automatically renews issued certificates. It calculates _when_ to
-renew a certificate based on the X.509 certificate's `duration` as well as the
-`renewBefore` value. The `renewBefore` value specifies _how long_ before expiry
-a certificate should be renewed.
+renew a certificate based Certificate's [`duration`][certspec] and
+[`renewBefore`][certspec] fields, for example:
+
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Certificate
+spec:
+  duration: 2160h          # 3 months
+  renewBefore: 720h        # 1 month
+```
+
+[certspec]: /docs/reference/api-docs/#cert-manager.io/v1.CertificateSpec
+
+The `renewBefore` value specifies _how long_ before expiry a certificate should
+be renewed. In the above example, the Certificate is expected to be renewed 1
+months prior to expiry:
+
+```diagram
+certificate                                        expected                   certificate
+  issued                                          reissuance                    expiry
+    +--------------------------------------------------+---------------------------+
+  1 Jan                                              1 Mar                      31 Mar
 
-The `spec.duration` and `spec.renewBefore` fields on a Certificate can be used
-as hints to the issuer for setting the `duration` and `renewBefore` values on a
-X.509 certificate.
 
-{{% alert title="Duration and renewBefore may be ignored" color="warning" %}}
+    <------------------------------------duration---------------------------------->
+                                         3 months
+
+                                                        <--------renewBefore------->
+                                                                   1 month
+```
+
+When set, the `duration` and `renewBefore` fields are given as hints to
```

Comment:
It is not spec.duration that is used when calculating renewal https://github.com/jetstack/cert-manager/blob/master/pkg/controller/certificates/util.go#L272-L291